File: C++/2165.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    string phase;
    getline(cin,phase);
    if(phase.length() <= 140){
        cout << "TWEET" << endl;
    }
    else{
        cout << "MUTE" << endl;
    }
    return 0;
}

File: Go/2165.go
package main

import (
    "fmt"
    "os"
    "bufio"
)

func main () {
    var phase string
    scanner := bufio.NewScanner(os.Stdin)
    scanner.Scan()
    phase = scanner.Text()
    if len(phase) <= 140{
        fmt.Println("TWEET")
    } else {
        fmt.Println("MUTE")
    }
}
